sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -601,15 +596,13 @@ def test_no_table_and_create_if_needed_and_no_schema(
     client.tables.Get.side_effect = HttpError(
         response={'status': '404'}, url='', content='')
     create_disposition = beam.io.BigQueryDisposition.CREATE_IF_NEEDED
-    with self.assertRaises(RuntimeError) as exn:
+    with self.assertRaisesRegexp(
+        RuntimeError, r'Table project:dataset.table requires a schema. None can'
+                      r' be inferred because the table does not exist.'):
```

Comment:
<!--new_thread; commit:4709161da347f4fa5eedbd7f3c09911b573fadac; resolved:0-->
Let's add `^` and `$` and escape the '.' characters throughout.

Reply:
done